<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BVDEncrypt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bvdencrypt.qrc">
    <normaloff>:/bvdencrypt/bvdfree.ico</normaloff>:/bvdencrypt/bvdfree.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="CFileSelect" name="fsInputFile" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>45</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Pass phrase</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lePassPhrase"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>User name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leUserName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Email address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leEmail"/>
    </item>
    <item>
     <widget class="CFileSelect" name="fsOutputFile" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>45</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbxWriteErrorLog">
      <property name="text">
       <string>Write validation error messages to a file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="CFileSelect" name="fsErrorFile" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>45</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>100</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>NML Requirements</string>
       </attribute>
       <widget class="QCheckBox" name="cbxNml">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Encrypt lab test references</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbxNmlQuotes">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>292</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Remove extraneous quotation marks</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>PCHS Requirements</string>
       </attribute>
       <widget class="QCheckBox" name="cbxPchs">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>241</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Omit parish numbers</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnEncrypt">
        <property name="text">
         <string>Encrypt file</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About_BVDEncrypt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_About_BVDEncrypt">
   <property name="text">
    <string>&amp;About BVDEncrypt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CFileSelect</class>
   <extends>QWidget</extends>
   <header>cfileselect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fsInputFile</tabstop>
  <tabstop>lePassPhrase</tabstop>
  <tabstop>leUserName</tabstop>
  <tabstop>leEmail</tabstop>
  <tabstop>fsOutputFile</tabstop>
  <tabstop>cbxWriteErrorLog</tabstop>
  <tabstop>fsErrorFile</tabstop>
  <tabstop>btnEncrypt</tabstop>
 </tabstops>
 <resources>
  <include location="bvdencrypt.qrc"/>
 </resources>
 <connections/>
</ui>
